### DISCUSSION TITLE
JSON auto-discovery?

### DISCUSSION THREAD

**Discussion by kidehen** [mastodon/mastodon#23172] [General]
Given the URL `https://mastodon.social/tags/csv` that denote an HTML document about the #CSV hashtag and `https://mastodon.social/tags/csv.json` that denotes a JSON document about the same hashtag, is there any reason why this alternative document type relationship isn't reflected in the <head/> section of the HTML doc using  <link rel="alternate" /> ?

Suggestion addition:
```
<head>
<link rel="alternate" href="https://mastodon.social/tags/csv.json" type="application/activity+json" />
</head>  
```

Benefit? 
User-agent discovery of the associated JSON document. 

I've moved this over to a [new feature related issue](https://github.com/mastodon/mastodon/issues/23174).

**Comment by mjankowski** [comment]
This was done in https://github.com/mastodon/mastodon/pull/33179
